Marlin/src/pins/stm32f4/pins_BTT_SKR_V2_0_REV_A.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (c) 2021 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (c) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#define BOARD_INFO_NAME "BTT SKR V2 Rev. A"
+
+#error "SKR V2 Rev. A requires modification or drivers may be damaged. See https://bit.ly/3t5d9JQ for more information. Comment out this line to continue."
```

Comment:
If it's confirmed to only affect TMC2209, we _could_ check for that here before throwing this error (using `HAS_DRIVER(TMC2209)`). On the other hand, I assume that TMC2209 drivers could still be harmed even if they are set to something else.

Is it only the Safe Power Protect check which is causing the driver to get burnt?

Reply:
"It depends" is the best answer as to which drivers are affected. BTT expanded the message to say any TMC could be affected, but I wanted the error message to be prominent for anyone compiling with this board.